css/src/tokens/themes.scss

```diff
@@ -34,12 +34,12 @@ $themes: (
 		'control-border': $palette-grey-80,
 		'control-border-bottom': $palette-grey-90,
 		'inline-code': $palette-grey-30,
-		'code-highlight-background': $palette-yellow-10,
+		'code-highlight-background': $palette-orange-70,
```

Comment:
Should this be changing to orange? Feels like this is a different case than the other "warning"-ish cases.

Reply:
I only moved the color to orange because the color itself seems closer to a shade of orange than yellow (to me). I can move it back to yellow if it doesn't make sense.

Reply:
Ah, no, that's probably fine then - resolving